packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -538,9 +562,32 @@ export default class Health extends Component {
 
   _getSrUrl = sr => `srs/${sr.id}`
 
+  _getDuplicatedMacAddresses = createCollectionWrapper(
+    createSelector(
+      () => this._getVifsByMac(),
+      vifsByMac => {
+        const duplicatedMacAddresses = []
+        for (const [macAddress, vifs] of Object.entries(vifsByMac)) {
+          if (vifs.length > 1) {
+            duplicatedMacAddresses.push(macAddress)
+          }
+        }
+        return duplicatedMacAddresses.sort()
+      }
+    )
+  )
+
   _getPoolPredicate = createSelector(
     createSelector(() => this.state.pools, resolveIds),
-    poolIds => (isEmpty(poolIds) ? undefined : item => includes(poolIds, item.$pool))
+    poolIds =>
+      isEmpty(poolIds)
+        ? undefined
+        : item => {
+          if (isArray(item)) {
+            return item.find(element => includes(poolIds, element.$pool)) !== undefined ? true : false
```

Comment:
Use `_.some` or `_.every` instead of `_.find`

Reply:
Also, `condition ? true : false` is equivalent to `condition` :)